model/job-management/src/main/proto/beam_job_api.proto

```diff
@@ -179,6 +183,17 @@ message JobState {
 }
 
 
+message GetJobMetricsRequest {
+  string job_id = 1; // (required)
+}
+
+// All metrics for a given job.  Runners may support one or the other or both.
+message GetJobMetricsResponse {
```

Comment:
I started with this, per https://s.apache.org/get-metrics-api.

Converting between this proto and the SDK reprs ([Java](https://github.com/apache/beam/pull/7934/files#diff-e722f484e12b515353e20532d3550465), [Python](https://github.com/apache/beam/pull/7934/files#diff-08a37e2757c9df3f0e1d0ba3e2121eecR328)) made it clear that we were actually introducing a proto representation of SDKs' `MetricResults`.

For example, `MetricResultsProtos.java` was originally `GetJobMetricsResponseUtils.java`:

```
public class GetJobMetricsResponseUtils {
  public static GetJobMetricsResponse   toProto(        MetricResults metricResults) { … }
  public static         MetricResults fromProto(GetJobMetricsResponse          resp) { … }
}
```

The signatures (and implementations) implied a broader purpose in ongoing discussions around metrics structures in the protos and SDKs. 

Framing it as just part of this RPC obfuscated that role, to me.

Reply:
OK, re-introduced the proto.

Reply:
Thanks. Should it live in `metrics.proto` though?

Reply:
It is really specific to a particular job, but I don't feel that strongly.

Reply:
sounds good